witness.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: witness
+  version: "0.8.1"
+  epoch: 0
+  description: Witness is a pluggable framework for software supply chain risk management. It automates, normalizes, and verifies software artifact provenance.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/in-toto/witness
+      tag: v${{package.version}}
+      expected-commit: a16c4f975a71a054b68ed0ef452deed0a9452d08
+
+  - runs: |
+      export CGO_ENABLED=0
```

Comment:
Rather than manually exporting CGO_ENABLED, should this just be exported as part of the environment; e.g.
```yaml
environment:
  environment:
    CGO_ENABLED: 0
```